doc/rst/source/changes.rst

```diff
@@ -4,6 +4,72 @@
 Changelog
 =========
 
+New Features in GMT 6.2
+=======================
+
+GMT 6.2 includes a new module, new common option, general code and documentation improvements, and numerous
+bug fixes! Here are the general updates included in 6.2:
+
+#. Addition of :doc:`theme-settings` (sets of GMT defaults), with a default modern theme for modern mode, and
+   :ref:`auto scaling options <cookbook/features:Automatic GMT settings>` for many GMT defaults.
+#. New :doc:`animation 13 <animations/anim13>` of seismic waveforms.
+#. Support for **+a**\ *angle* for y-axis as well as x-axis with the :ref:`-B axes settings <gmt:Axes settings>`.
+#. General improvements to the automatic determination of frame attributes.
+#. New **+d**\ *divisor* modifier to the :ref:`-i option <gmt:The **-i** option>` to simplify scaling of input values.
+#. Allow parsing of **-Jz**\ *1:zzzzzz* for vertical scale.
+#. New GMT configuration parameters :term:`MAP_FRAME_PERCENT`, :term:`COLOR_SET`, and :term:`COLOR_CPT`.
+#. Add support for reading variable in NetCDF-4 groups.
+#. Allow specifying the reciprocal increment for generating 1d arrays.
+#. Allow Latex expressions in single-line titles and Cartesian axes labels, add support for multi-line plot titles, and
+   add support for subtitles.
+#. Many general documentation improvements.
+#. API improvements to support the GMT/MEX, PyGMT, and GMT.jl environments.
```

Comment:
This is one is too vague and doesn't ring any bell to me. Perhaps drop it

Reply:
I updated with Paul's suggested phrasing.